includes/forms/class-wp-job-manager-form-submit-job.php

```diff
@@ -166,6 +166,22 @@ public function get_job_id() {
 		return absint( $this->job_id );
 	}
 
+	/**
+	 * Checks if it is forced application default field
+	 *
+	 * @return bool
+	 */
+	protected function is_forced_application_default_field() {
```

Comment:
This is a tough one to name, but I'm not sure this gets at what it does. I'm slightly leaning towards making this _just_ about validation and not sanitization. For sanitization they can use the `submit_job_form_fields` filter to change the `sanitizer`. In that case, we could move the filter down and name it something like `job_manager_application_field_use_standard_validation` (feel free to iterate on that one).

Reply:
Yes! It was very tough to name and really it's not so good.
I like your approach! It reduces the complexity \o/